ipynb/observationTable.md

```diff
@@ -0,0 +1,164 @@
+
+
+```python
+import pandas as pd
+import os.path
+```
+
+
+```python
+path_root = '/media/barnacle/projects/emp/'
+resultsFile = '/home/sjanssen/Desktop/observations.tsv'
```

Comment:
Is there a way we can make this more general, so that we can run this on different machines?